src/sso/sso.spec.ts

```diff
@@ -257,5 +257,29 @@ describe('SSO', () => {
         });
       });
     });
+
+    describe('getConnection', () => {
+      it(`requests a Connection`, async () => {
```

Comment:
@logangingerich Can you change this to single quotes instead of backticks (template strings)?